lib/ansible/config/base.yml

```diff
@@ -329,12 +329,20 @@ ACTION_WARNINGS:
   - {key: action_warnings, section: defaults}
   type: boolean
   version_added: "2.5"
+BECOME_PLUGIN_PATH:
+  name: Become plugins path
+  default: ~/.ansible/plugins/become:/usr/share/ansible/become/action
```

Comment:
`/usr/share/ansible/become/action` ?  should this be /usr/share/ansible/plugins/become ?

Reply:
good catch, bad edit after copy+paste